satellite/metainfo/metainfo.go

```diff
@@ -805,18 +824,25 @@ func (endpoint *Endpoint) SetBucketAttribution(context.Context, *pb.BucketSetAtt
 	return resp, status.Error(codes.Unimplemented, "not implemented")
 }
 
```

Comment:
**Rational for code Changes:**
added necessary changes to handle conversion from/to proto and storj.bucket